### ISSUE TITLE
Subscription ends_at timestamp nil

### ISSUE THREAD

**Issue by kouroubel** [pay-rails/pay#621]
I have successfully set up Stripe checkout in a test application and finally got the webhooks working as well. 

Is the Subscription ends_at timestamp supposed to be updated automatically or do I have to add an after_create method for that to work?

Here is what is stored in the database after the payment is completed.
![Subscription_first](https://user-images.githubusercontent.com/35150537/166660324-e10388c6-7dff-4126-b895-f403e4c0a0e8.jpg)

**Comment by kouroubel** [comment]
OK maybe I did not get the purpose of ends_at so let me rephrase the question. If the user buys a yearly subscription, let's say to a web app, how am I going to infer the subscription has expired, so I shutdown access to the app until subscription is renewed?

**Comment by kouroubel** [comment]
So if it is a yearly subscription which is created on 04 May 2022, why doesn't the ends_at automatically get the value of 04 May 2023?

**Comment by excid3** [comment]
Like I said, `ends_at` is the cancellation date. You would have to cancel the subscription to set it. 

This has nothing to do with the renewal schedule.
